server/registry/server_test.go

```diff
@@ -65,22 +65,21 @@ func defaultTestServer(t *testing.T) TestServer {
 	if hostedProject != "" {
 		p := remote.NewProxyForHostedService(hostedProject)
 		if err = p.Open(context.Background()); err != nil {
-			t.Fatalf("Fatalf: failed to connect to remote server: %s", err)
+			t.Fatalf("Setup: failed to connect to remote server: %s", err)
```

Comment:
Fixing this while I'm here, unsure what I was thinking when I put "Fatalf" there.